met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx

```diff
@@ -71,7 +71,7 @@ const DocumentOptionCard = () => {
                         spacing={1}
                     >
                         <Grid item>
-                            <DescriptionOutlinedIcon sx={{ pl: '25px', fontSize: '5em' }} />
+                            <DescriptionOutlinedIcon color={'info'} sx={{ pl: '25px', fontSize: '5em' }} />
```

Comment:
no need for brackets color='info' should work

Reply:
removed thank you for the suggestion